src/game_entities/foe.py

```diff
@@ -135,9 +136,15 @@ def roll_for_loot(self) -> Sequence[Item]:
 
     def get_formatted_keywords(self) -> str:
         """Return the list of keywords of the foe in a formatted version"""
-        return ", ".join(
-            [keyword.name.lower().capitalize() for keyword in self.keywords]
-        )
+        formatted_string: str = ""
+        for keyword in self.keywords:
+            try:
+                formatted_string += dict_foe_keywords[keyword.name.lower()] + ", "
+            except KeyError:
+                formatted_string += keyword.name.lower().capitalize()
```

Comment:
Huh where is the comma in this case?

Reply:
To seperate two keywords. For example "Undead, Large". It's the original method, but I forgot to add ` + ", "` at the end of Line144.